src/connection/index.js

```diff
@@ -33,22 +33,22 @@ export default class Connection {
      */
     getBlock(blockId) {
         return this._req(this.getApiUrls('blocks_detail'), {
-                urlTemplateSpec: {
-                    blockId
-                }
-            });
+            urlTemplateSpec: {
+                blockId
+            }
+        })
     }
 
     /**
      * @public
-     * @param tx_id
+     * @param txId
      */
-    getStatus(tx_id) {
+    getStatus(txId) {
         return this._req(this.getApiUrls('statuses'), {
-                query: {
-                    tx_id
-                }
-            });
+            query: {
+                txId
```

Comment:
not so sure if renaming `tx_id` to `txId` is without consequences here. Can someone confirm if that's alright? More parameters renaming further down this file with `block_id` -> `blockId`